tests/test_bgp_af.rb

```diff
@@ -829,6 +833,6 @@ def test_table_map
     afs.each do |af|
       table_map(55, 'red', af)
     end
-    config('no route-policy sjc') if platform == :ios_xr
+    config_no_warn('no route-policy sjc') if platform == :ios_xr
```

Comment:
Sounds good.  For a another day, might be better to move the creation of all test route_policies into ``config_ios_xr_dependencies``, and then create an ``unconfig_ios_xr_dependencies`` to remove them.  Seems like those methods could then be called in ``setup`` and ``teardown``.  A task for another PR, though.